civitai-downloader-v2/deployment/healthcheck.py

```
@@ -0,0 +1,50 @@
+try:
+    from src.core.reliability.health_check import HealthChecker, HealthStatus
+    from src.core.config.manager import ConfigManager
+except ImportError as e:
+    print(f"Failed to import health check modules: {e}")
+    sys.exit(1)
+
+async def main():
+    """Run health check."""
+    try:
+        # Initialize basic configuration
+        config_manager = ConfigManager()
```

Ruff (0.12.2)

Local variable `config_manager` is assigned to but never used

Remove assignment to unused variable `config_manager`

(F841)